pkg/cmd/attestation/verify/verify_integration_test.go

```diff
@@ -111,6 +111,25 @@ func TestVerifyIntegration(t *testing.T) {
 		require.Error(t, err)
 		require.ErrorContains(t, err, "verifying with issuer \"sigstore.dev\"")
 	})
+
+	t.Run("with bundle from OCI registry", func(t *testing.T) {
+		opts := Options{
+			APIClient:             api.NewLiveClient(hc, host, logger),
+			ArtifactPath:          "oci://ghcr.io/malancas/attest-demo:latest",
```

Comment:
Should we use a `ghcr.io` owner that is not tied to your user? Maybe the policy controller helm charts or something else in `ghcr.io/github`?

Reply:
Good idea, the policy controller helm charts would be perfect.